### PR TITLE
Add docs for Dremel 3D Printer

### PR DESCRIPTION
**Pull request by tkdrob**
## Proposed change

Add docs for Dremel 3D Printer

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/85969
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4424
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/dremel_3d_printer.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - Sensor
+ha_codeowners:
+  - '@tkdrob'
+ha_integration_type: device
+---
+
+The [Dremel 3D Printer](https://www.dremel.com/gn/en/digilab) can be added by knowing its Host IP address.
+
+.
+
```
Comment (frenck) [inline source/_integrations/dremel_3d_printer.markdown]:
```suggestion
```

File: source/_integrations/dremel_3d_printer.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ha_config_flow: true
+ha_domain: dremel_3d_printer
+ha_platforms:
+  - Sensor
+ha_codeowners:
+  - '@tkdrob'
+ha_integration_type: device
+---
+
+The [Dremel 3D Printer](https://www.dremel.com/gn/en/digilab) can be added by knowing its Host IP address.
```
Comment (frenck) [inline source/_integrations/dremel_3d_printer.markdown]: This is more of a prerequisite instead of an introduction.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
The PR for our brands repository is missing.

Could you create one @tkdrob and link it to this PR?

<https://github.com/home-assistant/brands>

Thanks already! 👍 

../Frenck

**Comment by tkdrob** [comment]
> The PR for our brands repository is missing.
> 
> Could you create one @tkdrob and link it to this PR?
> 
> https://github.com/home-assistant/brands
> 
> Thanks already! 👍
> 
> ../Frenck

Don't we already have a brand logo for Dremel? We should have one already.

Update: Guess I just need to move the logo out of custom components then.

File: source/_integrations/dremel_3d_printer.markdown [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: Dremel 3D Printer
+description: Instructions on how to integrate a Dremel 3D Printer into Home Assistant.
+ha_category:
+  - 3D Printing
+  - Sensor
+ha_iot_class: Local Polling
+ha_release: 2023.6
```
Comment (frenck) [inline source/_integrations/dremel_3d_printer.markdown]:
```suggestion
ha_release: 2023.7
```

**Review by frenck — APPROVED** [review]
Thanks, @tkdrob!

../Frenck

**Pull request merged**